pkg/drivers/kic/oci/cli_runner.go

```diff
@@ -84,6 +86,17 @@ func PrefixCmd(cmd *exec.Cmd) *exec.Cmd {
 	return cmd
 }
 
+func suppressDockerMessage() bool {
+	envKey := "MINIKUBE_SUPPRESS_DOCKER_PERFORMANCE"
+	env := os.Getenv(envKey)
+	suppress, err := strconv.ParseBool(env)
+	if err != nil {
+		klog.Errorf("failed to parse bool from the %s env, defaulting to 'false'; received: %s: %v", envKey, env, err)
```

Comment:
this should be a warning at worst, maybe even an info

Reply:
That's fair, my main concern was letting the user know that they messed by the config, but I can add an `out` message to satisfy that.

Reply:
Updated